roiextractors/extractors/suite2p/suite2psegmentationextractor.py

```diff
@@ -29,10 +29,11 @@ def __init__(self, fileloc, combined=False, plane_no=0):
         self.plane_no = plane_no
         self.filepath = fileloc
         self.stat = self._load_npy('stat.npy')
-        self.F = self._load_npy('F.npy', mmap_mode='r')
-        self.Fneu = self._load_npy('Fneu.npy', mmap_mode='r')
-        self.spks = self._load_npy('spks.npy', mmap_mode='r')
-        self.iscell = self._load_npy('iscell.npy', mmap_mode='r')
+        self._roi_response = self._load_npy('F.npy')
+        self._roi_response_fluorescence = self._roi_response
```

Comment:
does this need to be duplicated into two different attributes?

Reply:
fixed due to renaming and merging of other PRs